app/pb_kits/playbook/pb_selectable_card/_selectable_card.jsx

```diff
@@ -55,9 +51,6 @@ const SelectableCard = ({
   const dataProps = buildDataProps(data)
   const handleChange = event => {
```

Comment:
You don't need the `handleChange` anymore. You can simply add the `onChange` in the input.